packages/keyring-eth-ledger-bridge/src/ledger-error-handler.test.ts

```diff
@@ -0,0 +1,200 @@
+import { TransportStatusError } from '@ledgerhq/hw-transport';
+
+import { handleLedgerTransportError } from './ledger-error-handler';
+import { LedgerStatusError } from './type';
+
+describe('handleLedgerTransportError', () => {
+  const fallbackMessage = 'Default error message';
+
+  /**
+   * Helper function to create a TransportStatusError-like object
+   *
+   * @param message - The error message
+   * @param statusCode - The status code
+   * @returns A TransportStatusError instance
+   */
+  function createTransportStatusError(
+    message: string,
+    statusCode: number,
+  ): TransportStatusError {
+    const error = {
+      statusCode,
+      message,
+      name: 'TransportStatusError',
+    };
+    Object.setPrototypeOf(error, TransportStatusError.prototype);
+    return error as TransportStatusError;
+  }
+
+  describe('when error is TransportStatusError', () => {
+    it('should handle status code 0x6985 (user rejection)', () => {
```

Comment:
We normally don't use `should ` anymore, but we prefer the imperative forms for all TC now, could you update that as well?